dangerzone/conversion/pixels_to_pdf.py

```diff
@@ -146,7 +146,7 @@ async def convert(self) -> None:
         self.update_progress("Safe PDF created")
 
         # Move converted files into /safezone
-        if not running_on_qubes():
+        if not qubes_native_conversion():
```

Comment:
Same here, we should keep `running_on_qubes()`.

Reply:
done in [6ec1b0b](https://github.com/freedomofpress/dangerzone/pull/464/commits/6ec1b0b15e6824c34a039540d4e9331773c5253c)